bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -131,6 +131,12 @@ case class LastStepConverter(semester: Semester) extends SemesterConverter {
  * This converter will upgrade to 2017B
  */
 case object SemesterConverter2017ATo2017B extends SemesterConverter {
+  // TODO: REL-3079 remove this when the GN laser is repaired.
+  val altairLgsNotAvailable: TransformFunction = {
+    case a @ <altair>{ns @ _*}</altair> if (a \\ "lgs").nonEmpty =>
+      StepResult(s"Altair Laser Guidestar is currently not offered", a).successNel
```

Comment:
It doesn't actually remove the node at all: it just issues the message.
We want to keep the node in place for now and put an error in the problems section, preventing submission until the user manually fixes the problem.